Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -327,16 +327,15 @@ void AVIDump::AddFrame(const u8* data, int width, int height, int stride, const
     ERROR_LOG(VIDEO, "Error while encoding video: %d", error);
 }
 
-void AVIDump::Stop()
+static void HandleDelayedPackets()
```

Comment:
Can you squash this into the other delayed frames commit? It's easier read through that way.